sim-cli/src/main.rs

```diff
@@ -16,19 +16,35 @@ struct Cli {
     config: PathBuf,
     #[clap(long, short)]
     total_time: Option<u32>,
+    #[clap(long, short)]
+    debug: bool,
```

Comment:
Bool config option vs passing in a string that allows enabling other values like `trace`?